src/vs/editor/contrib/linesOperations/common/linesOperations.ts

```diff
@@ -689,17 +689,24 @@ export class TransposeAction extends EditorAction {
 	}
 
 	public run(accessor: ServicesAccessor, editor: ICommonCodeEditor): void {
+		for (let selection of editor.getSelections()) {
```

Comment:
Once we add this feature, maybe we should not call it `Transpose characters around the cursor ` any more.